src/coordsim/simulation/simulatorparams.py

```diff
@@ -7,11 +7,14 @@
 
 """
 import numpy as np
+import random
 
 
 class SimulatorParams:
     def __init__(self, network, ing_nodes, eg_nodes, sfc_list, sf_list, config, metrics, prediction=False,
                  schedule=None, sf_placement=None):
+        # Bool to store if simulator is called in warmup
+        self.warmup = True
```

Comment:
Just a leaving a pointer: Won't be needed once we fix https://github.com/RealVNF/rl-coordination/issues/199